infra/build.gradle.kts

```diff
@@ -44,22 +47,22 @@ dependencies {
     }
 
     // Actual deps
-    implementation("com.amazonaws:aws-java-sdk-cloudfront:1.12.459")
+    implementation("com.amazonaws:aws-java-sdk-cloudfront:1.12.560")
     implementation("org.bouncycastle:bcpkix-jdk15on:1.70")
-    implementation("org.springframework.boot:spring-boot-starter-actuator")
-    implementation("org.springframework.boot:spring-boot-starter-jdbc")
-    implementation("org.springframework.boot:spring-boot-starter-web")
-    implementation("org.springframework.boot:spring-boot-starter-log4j2")
-    implementation("org.springframework.boot:spring-boot-starter-cache")
-    implementation("org.springframework.boot:spring-boot-starter-security")
-    implementation("org.springframework.boot:spring-boot-starter-webflux")
-    implementation("org.jetbrains.kotlin:kotlin-reflect")
-    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
-    implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
-    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
-    implementation("com.zaxxer:HikariCP")
-    implementation("org.flywaydb:flyway-core:9.18.0")
-    implementation("com.github.ben-manes.caffeine:caffeine:3.1.6")
+    implementation("org.springframework.boot:spring-boot-starter-actuator:$springBootVersion")
```

Comment:
Hmm? Eihän näitä pitäisi tarvita kiinnittää tänne ollenkaan. Ne tulee spring dependency managementilta.

Reply:
Hmmjoo, IDEA ei lopettanut ulisemista siitä, että versiot menee miten sattuu. Palautan muutokset.

Reply:
Joo se on joskus vähän hidas tajuamaan. Gradle refresh + pieni odottelu kyllä yleensä saa sen tokenemaan.